### PR TITLE
Allow changing dev server's port

### PR DESCRIPTION
**Pull request by yonihemi**
Running `carton-release` also changed bundle.js's hash, so I just didn't commit the static resources.

### CODE REVIEW

File: entrypoint/dev.js [hunk 1]
```diff
@@ -33,7 +33,8 @@ const wasi = new WASI({
   },
 });
 
-const socket = new ReconnectingWebSocket("ws://127.0.0.1:8080/watcher");
+const port = document.currentScript.getAttribute("data-port") || 8080;
+const socket = new ReconnectingWebSocket(`ws://127.0.0.1:${port}/watcher`);
```
Comment (j-f1) [inline entrypoint/dev.js:37]: This should make passing the port unnecessary.
```suggestion
const socket = new ReconnectingWebSocket(`ws://${location.host}/watcher`);
```

Comment (MaxDesiatov) [reply entrypoint/dev.js:37]: I concur, let's use `location.host`, which means that passing the port to `indexPage` and `data-port` attribute won't be needed.

**Review by MaxDesiatov — CHANGES_REQUESTED** [review]
This is a great feature and I would love to merge this, but I think it would be simpler to use `location.host` as Jed suggested in his comment.

**Review by MaxDesiatov — APPROVED** [review]
Wonderful, thank you!

**Review by j-f1 — APPROVED** [review]

**Review by RayZhao1998 — APPROVED** [review]

**Pull request merged**